climakitae/data_loaders.py

```diff
@@ -32,19 +34,22 @@ def _open_and_concat(file_list, selections, ds_region):
     all_files = xr.Dataset()
     for one_file in file_list:
         data = cat[one_file].to_dask()
+        attributes = deepcopy(data.attrs)
         source_id = data.attrs["source_id"]
-        if selections.variable not in ("precipitation (total)", "wind 10m magnitude"):
+        if selections.variable not in ("Precipitation (total)", "wind 10m magnitude"):
             data = data[selections.variable]
-        elif selections.variable == "precipitation (total)":
-            pass
+        elif selections.variable == "Precipitation (total)":
+            data = data["RAINC"] + data["RAINNC"]
```

Comment:
What units attribute gets saved with this new variable?

Reply:
mm, same as for the two that are added together to get it -- just verified